internal/tfimport/importer/resources_test.go

```diff
@@ -75,3 +85,43 @@ func TestFromConfigValuesErr(t *testing.T) {
 		}
 	}
 }
+
+func TestSimpleImporter(t *testing.T) {
+	tests := []struct {
+		reqFields []string
+		tmpl      string
+		want      string
+	}{
+		// Empty.
+		{[]string{}, "", ""},
+
+		// Simple case, insert some template fields.
+		{[]string{"project", "name"}, "projects/{{.project}}/resources/{{.name}}", "projects/myproject/resources/myresourcename"},
+	}
+	for _, tc := range tests {
+		imp := &SimpleImporter{Fields: tc.reqFields, Tmpl: tc.tmpl}
+		got, _ := imp.ImportID(resourceChange, configs[0])
```

Comment:
check error?

Reply:
There's another set of tests for just if it errors. We don't expect it to fail here, should I still check err?

Reply:
I looked at the other test and it does check err in the expected-good case, so I added it here.

Reply:
Yeah, the idea is that even in the 'good' case there may be an error if you make an unexpected change so you want to fail the test there. In the 'errors' case if there was no error you would fail.